backend/src/main/java/com/ideahub/model/IdeaCollaboratorId.java

```diff
@@ -1,57 +1,25 @@
 package com.ideahub.model;
```

Comment:
Did you figure out what this class is used for? It was auto-generated by Hibernate, but I wasn't sure what for :P